### DISCUSSION TITLE
Random Songs from Playlist

### DISCUSSION THREAD

**Discussion by Gianuzzi** [kraxarn/spotify-qt#250] [Q&A] [answered]
Hi there! 
I'm curious about the 'Random' feature. When I select 'Random', does the playlist generated match what Spotify would create randomly, or does it use a different random generator?
Thanks!
PD.: I love this light client <3. My PC has several rendering problems, and Spotify-qt was the best solution I could find. Thanks (again)!

**Comment by kraxarn** [comment, marked as answer]
The tracks are randomized by the client playing the music itself, and not spotify-qt, as it really just enables the shuffle feature for you, so if you use an official client, it's Spotify's definition of random. The only exception is the first track, which is randomly selected by spotify-qt, but the rest is up to the client itself to choose.

Glad to hear you're liking it! ☺️

**Reply by Gianuzzi** [reply]
Thanks for the answer! :)
